app/controllers/assessment360_controller.rb

```diff
@@ -15,21 +15,19 @@ def all_students_all_reviews
     course = Course.find(params[:course_id])
     @assignments = course.assignments.reject(&:is_calibrated).reject {|a| a.participants.empty? }
     @course_participants = course.get_participants
-    if @course_participants.empty?
-      flash[:error] = "There is no course participant in course #{course.name}"
-      redirect_to(:back)
-    end
+    inspect_course_participants(@course_participants)
     # hashes for view
     @meta_review = {}
     @teammate_review = {}
     @teamed_count = {}
     # for course
     # eg. @overall_teammate_review_grades = {assgt_id1: 100, assgt_id2: 178, ...}
     # @overall_teammate_review_count = {assgt_id1: 1, assgt_id2: 2, ...}
-    %w[teammate meta].each do |type|
+    %w[teammate meta].each do |type| 
       instance_variable_set("@overall_#{type}_review_grades", {})
       instance_variable_set("@overall_#{type}_review_count", {})
     end
+    
```

Comment:
Ditto.  Whitespace change with no apparent impact to the code.  Don't confuse readers with empty changes.